### PR TITLE
Implement job for testing Compliance Operator on ROSA

### PR DESCRIPTION
**Pull request by rhmdnd**
We can use the rosa-aws-sts-hcp workflow to create ROSA clusters in CI,
which gives us the opportunity to run the Compliance Operator on top of
ROSA. Since ROSA is different from self-managed OpenShift, we should
exercise the operator on ROSA just like we would any other OpenShift
version.

### CODE REVIEW

**Comment by rhmdnd** [comment]
The following PRs will need to merge before we can land this:

https://github.com/openshift/release/pull/51154
https://github.com/ComplianceAsCode/compliance-operator/pull/515

File: ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml [hunk 1]
```diff
@@ -106,6 +114,25 @@ tests:
...
+    env:
+      BYO_OIDC: "true"
+      CHANNEL_GROUP: stable
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: 4.15.8
+      REGION: us-east-2
+    test:
+    - as: test-ocp4-pci-dss-node
+      cli: latest
+      commands: make e2e-roas
```
Comment (xiaojiey) [inline ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml:130]: seems a typo issue:
`      commands: make e2e-rosa`?

Comment (rhmdnd) [reply ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml:130]: Good question - I'm implementing that in https://github.com/ComplianceAsCode/compliance-operator/pull/515

**Comment by rhmdnd** [comment]
/pj-rehearse pull-ci-ComplianceAsCode-compliance-operator-master-e2e-rosa

**Comment by rhmdnd** [comment]
Cool - this is working as expected:

```
make: *** No rule to make target 'e2e-rosa'.  Stop.
```

That make target is getting introduced in https://github.com/ComplianceAsCode/compliance-operator/pull/515.

I'm going to ack the rehearsal since this piece is working and we can use it to test https://github.com/ComplianceAsCode/compliance-operator/pull/515.

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Pull request merged**